namespaces/blueprint-parse-result.md

```diff
@@ -0,0 +1,119 @@
+# Blueprint Parser Result Namespace
+
+This document defines elements needed for holding the result of parsing
+API Blueprint. This document uses [API Namespace][].
```

Comment:
So, once again, while this may be generic enough I had no ambition to make this work for any parser / adapter. The aim was really to transition from what we have in the current API Blueprint parse result / AST. I can drop the "blueprint" word here but please do let's not make this into "capture 'em all solution" at the moment. We need to stay on the ground. 

Note to self: When it exactly happened I have stopped being dreamer? :sake: :cloud: